src/sentry/conf/server.py

```diff
@@ -1504,6 +1504,8 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     "organizations:ds-sliding-window": False,
     # If true certain Slack messages will be escaped to prevent rendering markdown
     "organizations:slack-escape-messages": False,
+    # Enable crons product by default
+    "organizations:monitors": True,
```

Comment:
Can we just completely drop the flag?

Reply:
nvm, @davidenwang pointed out to me that we still disable it in our single tenant instances.

Reply:
Unfortunately not, we have to keep it to have it disabled in ST